packages/react/src/Button/ButtonBase.module.css

```diff
@@ -98,7 +98,7 @@
 
   /* Button layout */
 
-  & .ButtonContent {
+  & :where(.ButtonContent) {
```

Comment:
I suspect `.ButtonContent` and `.Label` don't need `where` 🤔 just a variant of `ButtonBase` right?

Reply:
@langermank honestly just did this to keep the specificity the same but have no idea if it's needed 🤷 

Specifically right now this would be: `:where(.ButtonBase) .ButtonContent` at `0,1,0` and without the `:where` it would be `0,2,0` with `.ButtonBase .ButtonContent`.

Would it be helpful if I did not include `:where()` for descendant selectors unless we see a problem during testing?

Reply:
Ahh okay, no, I think that's fair. Carry on!